packages/compass-e2e-tests/helpers/commands/navigate-to-collection-tab.js

```diff
@@ -0,0 +1,42 @@
+const Selectors = require('../selectors');
+
+module.exports = function (app) {
+  async function navigateToCollection(dbName, collectionName) {
+    const client = app.wrappedClient;
+
+    const headerSelector = `${Selectors.CollectionHeaderTitle}[title="${dbName}.${collectionName}"]`;
+    const collectionSelector = `${Selectors.SidebarCollection}[title="${dbName}.${collectionName}"]`;
+
+    // if the collection header is already visible we don't have to browse to the collection
+    if (await client.isExisting(headerSelector)) {
+      return;
+    }
+
+    // search for the collection and wait for the collection to be there and visible
+    await client.clickVisible(Selectors.SidebarFilterInput);
+    await client.setValue(Selectors.SidebarFilterInput, collectionName);
+    await client.waitForVisible(collectionSelector);
+
+    // click it and wait for the collection header to become visible
+    await client.click(collectionSelector);
+    await client.waitForVisible(headerSelector);
+  }
+
+  return async function navigateToCollectionTab(dbName, collectionName, tabName) {
```

Comment:
Tried to make this idempotent, quick, fail fast and easy to debug. But also maybe a pattern we can apply elsewhere. I figure almost all the tests we'll write will start off by getting to some collection tab and then go from there. The way it is written here will hopefully be reliable and useful going forward.